docs/formatV3.txt

```diff
@@ -591,6 +598,28 @@ These TOTP parameters fields have the following format:
                       default is 0. The format is that of a 40 bit (5 byte)
                       Password Safe time_t value as specified in Section 3.1.3.
 
+[30] The attachment parameters fields are listed below. If an entry contains
+an attachment the fields Att MediaType and Att Content must be present,
+the others are optional (default to empty/zero).
+
+    Att Title = User-friendly name for the attachment
+
+    Att MediaType = MIME type - text string describing Media Type, per
+      http://www.iana.org/assignments/media-types
+
+    Att FileName = Original file name - kept mainly for the extension, to support
+      applications that require this for proper viewing.
+
+    Att FileCreationTime, Att FileModificationTime, Att FileAccessTime =
+      Times associated with the attachment file at the time of attachment.
+      Application may try to restore these when exporting the attachment.
+      Representation is as described in Section 3.1.3.
+
+    Att Content = The binary representation of the attachment.
+      Note that this is limited to 4GB (2^32 bytes), which should suffice.
```

Comment:
So, no smaller practical limit on the file size?

Reply:
The 4GB limit mentioned is just the technical limit of the field, we could impose any arbitrary limit here, or maybe just an advice?

Are there any limits imposed by Android? On iOS there is a 120 MB limit for the password integration (used by other apps to query the psafe database), although it is pretty easy to add an option to just ignore (after decryption) the fields inside that integration since it doesn't need the attachments. 

I tried with a few large images on iOS and that seems to work ok, but still not preferable of course. Just to see if there are any performance issues, which seems not.

I am currently working on integrating attachment support into the iOS version (for V4 format, to be extended to V3 once the spec settles). This will allow a user to select an image from their device's photo library to import. The current code will offer the user to "optimize" the image as described here (https://support.huvisoft.dev/hc/en-us/articles/33413903586196-Optimizing-images-when-importing-in-your-safe).
Note that this is not released yet, so subject to change.

But that's the way I am thinking to keep the database small / manageable. Importing photos taken with the camera as-is will quickly get you a monster database.